pyapacheatlas.core.graphql.GraphQLException
===========================================

.. currentmodule:: pyapacheatlas.core.graphql

.. autoexception:: GraphQLException